lib/chai/core/assertions.js

```diff
@@ -136,10 +171,17 @@ module.exports = function (chai, _) {
   /**
    * ### .ordered
    *
-   * Sets the `ordered` flag, later used by the `members`
-   * assertions.
+   * Sets the `ordered` flag, causing all `.members` assertions that follow in
+   * the chain to require that members be in the same order.
+   *
+   *     expect([1, 2]).to.have.ordered.members([1, 2])
+   *       .but.not.have.ordered.members([2, 1]);
+   *
+   * If both the `contains` and `ordered` flags are set, the ordering begins
+   * with the first element in the target.
```

Comment:
Shouldn't it be the opposite? The ordering should follow the same order as the array given to the `include` assertion, right? I'm not sure it was just me that got it the wrong way but maybe this could be more clear.